api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java

```diff
@@ -764,8 +764,8 @@ public GlobalProperty getGlobalPropertyByUuid(String uuid) {
 	 * @return memory in mega bytes
 	 */
 	private String convertToMegaBytes(long bytes) {
-		int ONE_KILO_BYTE = 1024;
```

Comment:
Did you take a look at our convention for constants? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
Yes, but this variable wasn't made final by the author.

Reply:
does it change?

Reply:
Fixed :)